ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/auth/ServiceCallWithMainActorAuthTests.scala

```diff
@@ -37,4 +37,22 @@ trait ServiceCallWithMainActorAuthTests extends ServiceCallAuthTests {
   protected val canActAsMainActorExpiresTomorrow =
     Option(toHeader(expiringIn(Duration.ofDays(1), readWriteToken(mainActor))))
 
+  protected lazy val canReadAsMainActorActualLedgerId =
```

Comment:
Probably worth mentioning the reason behind these here as well, in case someone ends up here without context. :wink: